includes/Modules/Analytics/Measurement_Code_Injector.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Measurement_Code_Injector
+ *
+ * @package   Google\Site_Kit\Modules\Analytics
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics;
+
+use Google\Site_Kit\Context;
+
+/**
+ * Injects Javascript based on the current active plugins
+ *
+ * Class Injector
```

Comment:
Nit-picking, but like other class descriptions in Site Kit codebase this should rather just be a single line which describes the class (not starting with a verb) - e.g. "Class for injecting event tracking JavaScript based on currently active plugins.". The docblocks on other new classes should be adjusted similarly.